app/views/social_networking/profile_pages/show.html.erb

```diff
@@ -1,3 +1,8 @@
+<div ng-controller="AlertsCtrl as displayAlert">
```

Comment:
A more idiomatic Angular approach would probably be to use a directive here, especially if you don't intend to add any methods to the Alerts controller.